dbt_project.yml

```diff
@@ -1,6 +1,6 @@
 name: 'iterable'
 
-version: '0.13.1'
+version: '0.13.2'
```

Comment:
Bump to 0.14.0 and also update other versions and README mentions

Reply:
Updated.